package.json

```diff
@@ -35,6 +35,8 @@
     "build:dev:common": "lerna exec --stream --scope '*/*common' -- 'del-cli dist/** && tsc --project ./tsconfig.json && webpack --env=dev --target=node --config ../../webpack.config.js'",
     "test:cmd-api-server": "tap --ts --timeout=600 \"packages/cactus-*cmd-api-server/src/test/typescript/{unit,integration}/\"",
     "test:plugin-ledger-connector-besu": "tap --ts --jobs=1 --timeout=60 \"packages/cactus-*-besu/src/test/typescript/{unit,integration}/\"",
+    "test:odap": "tap --ts --timeout=600 \"packages/cactus-odap/src/test/typescript/{unit,integrate}/\"",
+    "build:dev:plugin-htlc-eth-besu-erc20": "lerna exec --stream --scope 'packages/*htlc-eth-besu-erc20' -- 'del-cli dist/** && tsc --project ./tsconfig.json && webpack --env=dev --target=node --config ../../webpack.config.js'",
```

Comment:
@jscode017 This one is no longer needed, the recommended way is now to use the watch script or the build scrpit which does caching on a package level anyway (so we are phasing out all the dedicated build scripts we have for building individual packages.)

Reply:
Ok, resolve with new commits